javaslang/src/main/java/javaslang/Value.java

```diff
@@ -516,7 +516,7 @@ default CharSeq toCharSeq() {
      */
     @SuppressWarnings("unchecked")
     @GwtIncompatible("reflection is not supported")
-    default T[] toJavaArray(Class<T> componentType) {
+    default T[] toJavaArray(Class<? super T> componentType) {
```

Comment:
@jbgi [found it](https://github.com/functionaljava/functionaljava/blob/master/core/src/test/java/fj/data/ArrayTest.java#L14-L19) :)

```java
    @Test
    public void array_is_safe() {
        List<Integer> list = List.range(1, 2);

        assertThat(list.toArray().array(Integer[].class), instanceOf(Integer[].class));
    }
```